src/main/java/org/gridsuite/merge/orchestrator/server/repositories/IgmRepository.java

```diff
@@ -19,25 +20,18 @@
  * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
  */
 @Repository
-public interface IgmRepository extends CassandraRepository<IgmEntity, IgmEntityKey> {
+public interface IgmRepository extends JpaRepository<IgmEntity, IgmEntityKey> {
 
-    @Query("SELECT * FROM merge_igm WHERE processUuid = :processUuid")
+    @Query(value = "SELECT igm from #{#entityName} as igm where igm.key.processUuid = :processUuid")
     List<IgmEntity> findByProcessUuid(UUID processUuid);
 
-    @Query("SELECT * FROM merge_igm WHERE processUuid = :processUuid AND date = :date")
+    @Query(value = "SELECT igm from #{#entityName} as igm where igm.key.processUuid = :processUuid and igm.key.date = :date")
```

Comment:
Really necessary ?

Reply:
primary key is composite key

Reply:
See above

Reply:
Done in PR for migration to PostGres : https://github.com/gridsuite/merge-orchestrator-server/pull/50